lib/line-numbers.js

```diff
@@ -17,8 +17,8 @@ module.exports = {
 };
 
 const NUMBER_REGEX = /^-?\d+$/;
-const RANGE_REGEX = /^(-?\d+)-(-?\d+)$/;
-const LINE_NUMBERS_REGEX = /^(-?\d+(-\d+)?,?)+$/;
+const RANGE_REGEX = /^(?<startGroup>-?\d+)-(?<endGroup>-?\d+)$/;
```

Comment:
I did use named capturing groups at first, but you (and the integration tests) told me not to because of Node version compatibility? Has that changed?

Does the same apply to the `Array.flat()` and `Object.fromEntries()` which I also replaced by request?

Reply:
I see. Thank you.